packages/ckeditor5-code-block/tests/codeblockui.js

```diff
@@ -197,6 +197,14 @@ describe( 'CodeBlockUI', () => {
 				expect( button ).to.have.property( 'isToggleable', true );
 			} );
 
+			it( 'and it\'s arrow has the base properties', () => {
```

Comment:
> and it\\'s arrow

There should be not and as this unit test doesn't test anything about the button (it's the purpose of the previous "button has the base properties" unit test.